### ISSUE TITLE
Computed based on an observable returns undefined the first time it's read

### ISSUE THREAD

**Issue by axtgr** [LegendApp/legend-state#199]
```
const obs = observable(1);
const comp = computed(() => obs)

console.log(obs.get()) // 1
console.log(comp.get()) // undefined
console.log(comp.get()) // 1
```

This isn't supposed to happen, right?

**Comment by jmeistrich** [comment]
Nope, that's not right! I'll look into it.

**Comment by jmeistrich** [comment]
This is fixed in version `2.0.0-beta.2`. Thanks for the report!

**Comment by axtgr** [comment]
Thanks for the quick fix!
